bin/inference.py

```diff
@@ -26,8 +26,8 @@
     hparams = lightning_module.hparams["hparams"]
 
     # Get dataset for inference
-    _, _, test_ds = get_datasets(
-        hparams.dataset_name,
+    infer_ds, _, _ = get_datasets(
+        "poesy_infer",
```

Comment:
Be careful here you have hardcoded the dataset name, this means we cannot do titan inference anymore. Also you changed the index of inference dataset (test in our case). Could you fix that ?

Reply:
Ok for the index, thanks for noticing it. But I have a question about the dataset name. Currently, I have modified only the dataset Poesy (that is why it is hardcoded). If we want to preserve the proper functioning of the inference on Titan, I should modifiy it as well.
Is it okay for you if I try to modify it the same way I modify Poesy ? It means to create TitanInferDataset and InferSample. And also to dissociate num_pred_steps variable in TitanSettings as num_output_steps and num_inference_pred_steps.

Reply:
@CorentinSeznec let's not modify TITAN in this PR I would like to ask @LBerth first. If you could fix the index and restore the name arg passing in this PR I can merge it and we see next week what we do about TITAN. Thanks !

Reply:
@colon3ltocard Ok, it is done. Thank you !